custom/icds_reports/static/icds_reports/js/spec/awcs-covered.directive.spec.js

```diff
@@ -218,14 +218,14 @@ describe('AWCs Covered Directive', function () {
         controller.userLocationId = 'test_id4';
         controller.location = {name: 'name4', location_id: 'test_id4'};
         controller.selectedLocations.push(
-            {name: 'name1', location_id: 'test_id1'},
-            {name: 'name2', location_id: 'test_id2'},
-            {name: 'name3', location_id: 'test_id3'},
-            {name: 'name4', location_id: 'test_id4'},
-            {name: 'name5', location_id: 'test_id5'},
-            {name: 'name6', location_id: 'test_id6'}
+            {name: 'name1', location_id: 'test_id1', user_have_access: 0},
```

Comment:
What do these locations coorespond to? are they states, districts, blocks etc?

Reply:
@emord In ```selectedLocation``` variable we keep the whole selected path, where the first item in the list represents selected state, second district etc.